.vscode/launch.json

```diff
@@ -12,7 +12,7 @@
             "console": "integratedTerminal"
         },
         {
-            "name": "Pytest: Current File",
+            "name": "Debug Tests: Current File",
```

Comment:
Nice :) 

Can we also add the `justMyCode`: false and also the `envFile`, `"--no-cov"` setting etc. to the `"Debug Tests"`?

I think these are nice settings your introduced.